x-pack/plugins/cloud_security_posture/common/constants.ts

```diff
@@ -8,3 +8,8 @@
 export const CSP_KUBEBEAT_INDEX_PATTERN = 'kubebeat*';
 export const CSP_KUBEBEAT_INDEX_NAME = 'findings';
 export const STATS_ROUTH_PATH = '/api/csp/stats';
+export const FINDINGS_ROUTH_PATH = '/api/csp/findingstmp'; // TODO: update when we switch with FE
+export const AGENT_LOGS_INDEX = 'kubebeat*';
+export const DEFAULT_FINDINGS_PER_PAGE = 20;
+export const RULE_PASSED = `passed` as const;
+export const RULE_FAILED = `failed` as const;
```

Comment:
```suggestion
export const RULE_PASSED = "passed"
export const RULE_FAILED = "failed"
```
no need for casting `as const` for literals

Reply:
fixed